lib/sidekiq/scheduler.rb

```diff
@@ -140,7 +140,7 @@ def self.enqueue_job(job_config)
       config['class'] = config['class'].constantize if config['class'].is_a?(String)
       config['args'] = Array(config['args'])
 
-      if defined?(ActiveJob) && config['class'].included_modules.include?(ActiveJob::Enqueuing)
+      if defined?(ActiveJob::Enqueuing) && config['class'].included_modules.include?(ActiveJob::Enqueuing)
```

Comment:
This could be used on non-rails apps. I think this will not work on non-rails app maybe we should add an extra check. What do you think?